packages/graphiql-plugin-explorer/src/graphiql-explorer.d.ts

```diff
@@ -52,7 +52,7 @@ declare module 'graphiql-explorer' {
       buttonStyle?: CSSProperties;
       actionButtonStyle?: CSSProperties;
     } | null;
-    showAttribution: boolean;
+    showAttribution?: boolean;
```

Comment:
I did a global search for this prop but couldn't really find anything so sadly I will admit I just made it optional without fully understanding the implications 😓

Reply:
i think it's fine, as these are manually maintained types for the external `graphql-explorer` flow library, I can show a PR where i convert it to ts and adopt it into the repo